apis/conversion/src/__tests__/index.test.js

```diff
@@ -163,16 +170,55 @@ describe('objectConversion', () => {
       expect(supernovaMock).toHaveBeenCalledTimes(1);
       expect(exportPropertiesMock).toHaveBeenCalledTimes(0);
       expect(sourceQaeExportPropertiesMock).toHaveBeenCalledTimes(1);
-      expect(sourceQaeExportPropertiesMock).toHaveBeenCalledWith({ propertyTree, hypercubePath: '' });
+      expect(sourceQaeExportPropertiesMock).toHaveBeenCalledWith({
+        propertyTree,
+        hypercubePath: '',
+        viewDataMode: false,
+      });
       expect(importPropertiesMock).toHaveBeenCalledTimes(0);
       expect(targetQaeImportPropertiesMock).toHaveBeenCalledTimes(1);
       expect(targetQaeImportPropertiesMock).toHaveBeenCalledWith({
         exportFormat: { qProperty: { prop3: 'value 3' } },
         initialProperties: { qInfo: { qType: 'new type' }, visualization: 'new type' },
         dataDefinition: {},
         hypercubePath: '',
+        viewDataMode: false,
       });
       expect(newPropertyTree).toEqual({ qProperty: { prop4: 'value 4' } });
     });
+
+    test('should call export properties with viewdatamode', async () => {
```

Comment:
Now this is really important ;)
```suggestion
    test('should call export properties with viewDataMode', async () => {
```